### PR TITLE
fix: Self monitor load test

### PR DESCRIPTION
**Pull request by hisarbalik**
## Description

Changes proposed in this pull request (what was done and why):

- Fix flaky head chunk storage test, add wal storage size metric to the result
- Update docs for test result of self-monitor

Changes refer to particular issues, PRs or documents:

- 

## Traceability
- [ ] The PR is linked to a GitHub issue.
- [ ] The PR has a milestone set.
- [x] The PR has a respective `area` and `kind` label.
- [ ] The follow-up issues (if any) are linked in the `Related Issues` section.
- [ ] If the change is user-facing, the documentation has been adjusted.
- [ ] The feature is unit-tested.
- [ ] The feature is e2e-tested.

### CODE REVIEW

**Review by chrkl — APPROVED** [review]

File: docs/contributor/benchmarks/README.md [hunk 1]
```diff
@@ -349,14 +349,15 @@ Configured memory, CPU limits, and storage are based on this base value and will
 
 
 #### Test Results
-
+The main KPIs to tracks performance changes are Scrape Samples/sec and Total Series Created, this values should be in the range of 15-22 samples/sec and 200-350 series respectively.
```
Comment (NHingerl) [inline docs/contributor/benchmarks/README.md:352]:
```suggestion
The main KPIs to track performance changes are **scrape samples per sec** and _total series created_. These values should be in the range of 15-22 samples/sec and 200-350 series, respectively. 
```

File: docs/contributor/benchmarks/README.md [hunk 2]
```diff
@@ -349,14 +349,15 @@ Configured memory, CPU limits, and storage are based on this base value and will
 
 
 #### Test Results
-
+The main KPIs to tracks performance changes are Scrape Samples/sec and Total Series Created, this values should be in the range of 15-22 samples/sec and 200-350 series respectively. 
+Another metrics to track are CPU and memory usage of the self-monitor pods, both are directly influenced by the number of series created and the scrape samples/sec, more samples and series created will increase the memory and CPU usage of the self-monitor pods.
```
Comment (NHingerl) [inline docs/contributor/benchmarks/README.md:353]:
```suggestion
Other metrics to track are _CPU_ and _memory usage_ of the self-monitor Pods. Both are directly influenced by the number of series created and the scrape samples/sec: more samples and series created increase the memory and CPU usage of the self-monitor Pods.
```

Comment (hisarbalik) [reply docs/contributor/benchmarks/README.md]: changed manually with merging with main

File: docs/contributor/benchmarks/README.md (hunk 1 above)
Comment (hisarbalik) [reply docs/contributor/benchmarks/README.md]: changed manually with merging with main

File: docs/contributor/benchmarks/README.md [hunk 3]
```diff
@@ -350,15 +349,16 @@ Configured memory, CPU limits, and storage are based on this base value and will
 
 
 #### Test Results
-The main KPIs to tracks performance changes are Scrape Samples/sec and Total Series Created, this values should be in the range of 15-22 samples/sec and 200-350 series respectively. 
-Another metrics to track are CPU and memory usage of the self-monitor pods, both are directly influenced by the number of series created and the scrape samples/sec, more samples and series created will increase the memory and CPU usage of the self-monitor pods.
-<div class="table-wrapper" markdown="block">
+The main KPIs to track performance changes are **scrape samples per sec** and _total series created_. These values should be in the range of 15-22 samples/sec and 200-350 series, respectively.
```
Comment (NHingerl) [inline docs/contributor/benchmarks/README.md:352]:
```suggestion
The main KPIs to track performance changes are **scrape samples per sec** and **total series created**. These values should be in the range of 15-22 samples/sec and 200-350 series, respectively.
```
I missed this yesterday

File: docs/contributor/benchmarks/README.md [hunk 4]
```diff
@@ -350,15 +349,16 @@ Configured memory, CPU limits, and storage are based on this base value and will
...
+Other metrics to track are _CPU_ and _memory usage_ of the self-monitor Pods. Both are directly influenced by the number of series created and the scrape samples/sec: more samples and series created increase the memory and CPU usage of the self-monitor Pods.
```
Comment (NHingerl) [inline docs/contributor/benchmarks/README.md:353]:
```suggestion
Other metrics to track are **CPU** and **memory usage** of the self-monitor Pods. Both are directly influenced by the number of series created and the scrape samples/sec: more samples and series created increase the memory and CPU usage of the self-monitor Pods.
```

**Review by chrkl — APPROVED** [review]

**Review by NHingerl — APPROVED** [review]

**Pull request merged**